src/Uno.UI.Toolkit/Uno.UI.Toolkit.Windows.csproj

```diff
@@ -65,8 +65,8 @@
 
 		<PropertyGroup>
 			<_OverrideTargetFramework>$(TargetFramework)</_OverrideTargetFramework>
-			<_OverrideTargetFramework Condition="'$(_OverrideTargetFramework)' == 'uap10.0.19041'">UAP</_OverrideTargetFramework>
```

Comment:
This is not very important as we will rarely need it, but `UAP` looks correct capitalized. That's the casing we have in our package.